src/transformState.js

```diff
@@ -62,7 +62,27 @@
  * @param {Object[]} transforms
  */
 function transformState(state, transforms) {
-  // write code here
+  for (let i = 0; i < transforms.length; i++) {
+    if (transforms[i].operation === 'addProperties') {
```

Comment:
Instead of 'if... else if... else if...' use 'switch...case' statement. It's more appropriate in such kind of tasks